huaweicloud/services/elb/resource_huaweicloud_elb_logtanks.go

```diff
@@ -0,0 +1,144 @@
+package elb
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/chnsz/golangsdk/openstack/elb/v3/logtanks"
```

Comment:
`chnsz` is the self-developed repository, followed by `hashicorp`.

Reply:
this function is still developed in this package, so we still need to rely on this package

Reply:
sorry, I mean these packages needs to reorder.

Reply:
done